wremnants/datasets/datagroups.py

```diff
@@ -315,14 +329,16 @@ def histName(self, baseName, procName, syst):
             return syst
         return "_".join([baseName,syst])
     
-    def readHist(self, baseName, proc, syst, scaleOp=None, forceNonzero=True):
+    def readHist(self, baseName, proc, syst, scaleOp=None, forceNonzero=True, scaleToNewLumi=-1):
         output = self.results[proc.name]["output"]
         histname = self.histName(baseName, proc.name, syst)
         if histname not in output:
             raise ValueError(f"Histogram {histname} not found for process {proc.name}")
         h = output[histname]
         if forceNonzero:
             h = hh.clipNegativeVals(h)
+        if scaleToNewLumi > 0:
```

Comment:
Remind me what this does differently than just setting a different lumi value? It's to assume the relative stat uncertainty stays the same?

Reply:
it scales yields and variances by N (thus uncertainties by sqrt(N)) to simulate having more data or MC, while if you just scale the lumi weight you are left with the same relative statistical uncertainty